content/post/2021-12-01-mlr3spatial/index.Rmd

```diff
@@ -0,0 +1,30 @@
+---
+title: mlr3spatial
+author: Marc Becker
+date: '2021-12-01'
+slug: mlr3spatial
+categories:
+  - R
+tags:
+  - mlr3spatial
+  - CRAN
+---
+
+We are happy to announce that [mlr3spatial](https://mlr3spatial.mlr-org.com/) has been released on [CRAN](https://cran.r-project.org/web/packages/mlr3spatial/index.html). 
+[mlr3spatial](https://mlr3spatial.mlr-org.com/) simplifies the handling of spatial objects in the [mlr3](https://github.com/mlr-org) ecosystem.
+Before [mlr3spatial](https://mlr3spatial.mlr-org.com/), the user had to extract raw values from spatial objects to train a model or predict a map.
```

Comment:
"predict a map" 

-> We are not really predicting a map, we're creating a map from the predicted values which also inherit spatial information.
Also we might need to explain what "raw values" are here.

Reply:
> Also we might need to explain what "raw values" are here.

The user had to create tabular data from spatial objects to train a model or predict a map.

?

> We are not really predicting a map, we're creating a map from the predicted values which also inherit spatial information.

Sure that's technically correct but it's much simpler to just say map. Everybody knows  what a map is. But you are right. I change it in the other sentences.

Reply:
> The user had to create tabular data from spatial objects to train a model or predict a map.

Sounds better 👍 

> Sure that's technically correct but it's much simpler to just say map. Everybody knows what a map is. But you are right. I change it in the other sentences.

I know this sounds a bit nit-picky but keep in mind the audience: not only spatial people will read it and even if, being on point with terminology helps to convince them :)